src/core/types/decimal.rs

```diff
@@ -11,26 +11,26 @@ use radix_engine_common::math::Decimal as ScryptoDecimal;
     DeserializeFromStr,
     uniffi::Record,
 )]
-pub struct Decimal {
+pub struct Decimal192 {
```

Comment:
@CyonAlexRDX Why is this needed?

Reply:
Clash with Swift Foundations type with same name. So in iOS app in a file where we import
Both Sargon and Foundation it does not compile. And I don't wanna have to write Sargon.Decimal all over the place.